CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## 4.0.1-dev
 
 * Enforce lint rules found in pedantic 1.11.0.
+* Enforce `avoid_dynamic_calls` lint rule.
```

Comment:
🤷  I'm not a big believer in documenting changes that don't affect end-users in the chaneglog.

Reply:
Done.